internal/tasks/subscriptiontask.go

```diff
@@ -209,6 +210,9 @@ func (sm *SubscriptionTask) handleNodeBindingUpdates(ctx context.Context, newBin
 
 		if !found {
 			logger.Ctx(ctx).Infow("binding added", "key", newBinding.Key())
+			if !newBinding.ScheduledAt.IsZero() {
```

Comment:
This will only capture time to trigger a consul watch, we should take a diff with subscription updated_at to get the actual latency